presto-docs/src/main/sphinx/functions/qdigest.rst

```diff
@@ -30,6 +30,11 @@ Functions
     Returns the approximate percentile values from the quantile digest given
     the number ``quantile`` between 0 and 1.
 
+.. function:: scale_qdigest(qdigest(T), scale_factor) -> T
```

Comment:
Shouldn't the return type be qdigest(T)?

Reply:
Yep, fixed.